pkg/schema/migrate/v3.0.0.go

```diff
@@ -97,3 +100,30 @@ func rewriteHealthCheck30(ctx context.Context, session gocqlx.Session, ev migrat
 
 	return nil
 }
+
+func setExistingTasksDeleted(ctx context.Context, session gocqlx.Session, ev migrate.CallbackEvent, name string) error {
```

Comment:
Why do we need it?

Reply:
As discussed, keeping this so that we can query for all not-deleted tasks.